core/tests/test_db.py

```diff
@@ -1,9 +1,9 @@
 import pytest
 
 
-from crop.constants import SQL_CONNECTION_STRING
+from core.crop.constants import SQL_CONNECTION_STRING
 
-from crop.db import create_database, connect_db, drop_db, check_database_structure
+from core.crop.db import create_database, connect_db, drop_db, check_database_structure
```

Comment:
Instead of adding **core.**, could you please adjust your python path since we have restructured the repository.